daemons/gptp/common/ptp_message.cpp

```diff
@@ -1364,18 +1366,19 @@ void PTPMessagePathDelayRespFollowUp::processMessage(IEEE1588Port * port)
 	 * than a certain number of lost FUPs we should set asCapable
 	 * to false
 	 */
+	diffLastSeqId = sequenceId - port->getLastSeqId();
```

Comment:
So, as Andrew already mentioned, I understand by the code above and your (really nice) explanation about it that switching the unsigned int to a uint16_t would solve this problem. Is that correct? I will push in the diff already so if it is the case, we can move forward and integrate this